.github/workflows/helm-release.yaml

```
@@ -58,8 +58,8 @@ jobs:
             chartName: logshipper
           - chartDir: openbao/charts/openbao
             chartName: openbao
-          - chartDir: opentelemetry/chart
```

Could we rename this folder from `chart` to `charts`? As all the other plugins has naming convention `<plugin-name>/charts` It will be easier to do any kind of automation if all the plugins have the same naming convention :) 
This comment also applies to `audit-logs` plugin. 
@olandr
```suggestion
          - chartDir: opentelemetry/charts
```